charts/prometheus/templates/prometheus-alerts-configmap.yaml

```diff
@@ -112,107 +121,769 @@ data:
           for: 5m
           labels:
             tier: platform
-            severity: urgent
+            severity: critical
```

Comment:
Reviewing this alert based on your comment, both the `ManyUnhealthySchedulers` and `SchedulersNotHealthy` are looking for multiple unhealthy schedulers. Does it make sense to know if any amount of schedulers are unhealthy? Basically `unhealthschedulers > 0` vs the 50% or >=5 they are now?

Reply:
It does make sense, with the appropriate "for" configuration (because they go down momentarily when restarting i.e. how the panel looks 'spiky'). Basically anything besides 100% availability on schedulers should be actionable. However, if just a few are down, that is probably user error, and the action is to contact the customer and work through it with them. I am interested what you think about another alert category that is managed by customer support for this kind of customer-support-managed alert. I think you were saying to me earlier that it's part of the story for the prom operator, which makes it easier to add new alerts. Also, customer support needs a load off before they have any ability to jump in on something new.

Reply:
>with the appropriate "for" configuration (because they go down momentarily when restarting i.e. how the panel looks 'spiky').

What do you think would be the "for" here? Your gut on this is better than mine.

> I am interested what you think about another alert category that is managed by customer support for this kind of customer-support-managed alert.

I'm all for this, maybe give those alerts a "support" label or something then we make a corresponding alert manager route to send to their slack channel or wherever. The prom operator does make this easier in that they can own configmaps that container just their stuff and modify as needed. However, you still need to write promql. I don't think it would be a  big deal for us to do it for them until they want to/are ready to